backend/fms_core/serializers.py

```diff
@@ -452,7 +452,7 @@ class Meta:
                   'location_barcode', 'location_coord', 'container_full_location', 'site',
                   'current_volume', 'concentration', 'creation_date', 'collection_site', 'experimental_group',
                   'individual_name', 'sex', 'taxon', 'cohort', 'pedigree', 'father_name', 'mother_name',
-                  'quality_flag', 'quantity_flag', 'projects', 'depleted', 'is_library', 'derived_samples_counts', 'comment')
+                  'quality_flag', 'quantity_flag', 'project', 'depleted', 'is_library', 'derived_samples_counts', 'comment')
```

Comment:
I see that sample has `projects` property but not `project`. So not sure if this is incorrect or not.